### PR TITLE
dentclinic source

### PR DESCRIPTION
**Pull request by RudyiVT**

### CODE REVIEW

File: airbyte-integrations/connectors/source-dentclinic/source_dentclinic/source.py [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+import pendulum
+import xmltodict
+from airbyte_cdk.sources import AbstractSource
+from airbyte_cdk.sources.streams import Stream
+from airbyte_cdk.sources.streams.http import HttpStream
+
+from .streams import DentclinicBookingStream, DentclinicBookingFrStream, DentclinicClinicIdsStream, DentclinicStaticStream
+
+
+class BookingsFr(DentclinicBookingStream):
```
Comment (sebastianflaen) [inline airbyte-integrations/connectors/source-dentclinic/source_dentclinic/source.py:19]: Think we need to use DentclinicBookingFrStream instead with fr_days_forward and fr_days_back configd

File: airbyte-integrations/connectors/source-dentclinic/source_dentclinic/streams.py [hunk 2]
```diff
@@ -0,0 +1,395 @@
...
+# Basic full refresh stream
+class DentclinicBookingStream(HttpStream, ABC):
+    primary_key = None
+    state_checkpoint_interval = 1
+
+    url_base = "https://dcm-nhn.dentclinicmanager.com/API/DCMConnect.asmx?WSDL"
+
+    def __init__(self, config: Mapping[str, Any], **kwargs):
+        super().__init__()
+        self.fetch_interval_days = 1
```
Comment (sebastianflaen) [inline airbyte-integrations/connectors/source-dentclinic/source_dentclinic/streams.py:21]: Could we set this as a config?

File: airbyte-integrations/connectors/source-dentclinic/source_dentclinic/source.py [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+        """
+        return True, None
+
+    def streams(self, config: Mapping[str, Any]) -> List[Stream]:
+        """
+        :param config: A Mapping of the user input configuration as defined in the connector spec.
+        """
+        return [BookingsFr(config=config), Bookings(config=config)]
+
+        # ,Clinics(config=config), Services(config=config), Resources(config=config), BookingsFr(config=config)
```
Comment (sebastianflaen) [inline airbyte-integrations/connectors/source-dentclinic/source_dentclinic/source.py:144]: Didnt those work?

**Review by sebastianflaen — CHANGES_REQUESTED** [review]

File: airbyte-integrations/connectors/source-dentclinic/source_dentclinic/source.py (hunk 3 above)
Comment (RudyiVT) [reply airbyte-integrations/connectors/source-dentclinic/source_dentclinic/source.py:144]: No(

File: airbyte-integrations/connectors/source-dentclinic/source_dentclinic/streams.py (hunk 2 above)
Comment (RudyiVT) [reply airbyte-integrations/connectors/source-dentclinic/source_dentclinic/streams.py:21]: +

File: airbyte-integrations/connectors/source-dentclinic/source_dentclinic/source.py (hunk 1 above)
Comment (RudyiVT) [reply airbyte-integrations/connectors/source-dentclinic/source_dentclinic/source.py:19]: Added `days_back` parameter to config

File: airbyte-integrations/connectors/source-dentclinic/source_dentclinic/source.py (hunk 3 above)
Comment (RudyiVT) [reply airbyte-integrations/connectors/source-dentclinic/source_dentclinic/source.py]: Fixed now

**Review by sebastianflaen — APPROVED** [review]

**Pull request merged**